From 9202f1c906c375c28fe50070c326f770f750be67 Mon Sep 17 00:00:00 2001
From: Luciano Santos <luc14n0@opensuse.org>
Date: Tue, 18 Oct 2022 05:36:43 -0300
Subject: [PATCH] Replace deprecated attr/xattr.h with sys/xattr.h

We should move away from attr/xattr.h (libattr1) since it has been
deprecated for quite a while now and glibc's sys/xattr.h is a drop-in
replacement for it.
---
 meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index 5a7c7f6..aea2fb9 100644
--- a/meson.build
+++ b/meson.build
@@ -41,8 +41,8 @@ conf_data.set('HAVE_SYS_ACL_H', has_sys_acl_h)
 has_acl_libacl_h = compiler.has_header('acl/libacl.h')
 conf_data.set('HAVE_ACL_LIBACL_H', has_acl_libacl_h)
 
-has_xattr_h = compiler.has_header('attr/xattr.h')
-conf_data.set('HAVE_ATTR_XATTR_H', has_xattr_h)
+has_sys_xattr_h = compiler.has_header('sys/xattr.h')
+conf_data.set('HAVE_SYS_XATTR_H', has_sys_xattr_h)
 
 user_attributes_feature = get_option('user-attributes')
 conf_data.set('ENABLE_USER_XATTR',
-- 
2.37.3

